java/system-test/system-test-drivers/src/main/java/sleeper/systemtest/drivers/metrics/AwsTableMetricsDriver.java

```diff
@@ -59,11 +59,11 @@
 public class AwsTableMetricsDriver implements TableMetricsDriver {
     private static final Logger LOGGER = LoggerFactory.getLogger(AwsTableMetricsDriver.class);
     private static final Map<String, String> METRIC_ID_TO_NAME = Map.of(
-            "activeFiles", "ActiveFileCount",
+            "activeFiles", "NumberOfFilesWithReferences",
             "records", "RecordCount",
             "partitions", "PartitionCount",
             "leafPartitions", "LeafPartitionCount",
-            "filesPerPartition", "AverageActiveFilesPerPartition");
+            "filesPerPartition", "AverageFileReferencesPerPartition");
```

Comment:
The map keys can be updated for both filesPerPartition and activeFiles. They're just used to reference the metrics below because the API requires you to set an ID when you query for the metrics.

Reply:
Updated